Fix #6 and add myself to the footer :D

LukyVj opened the pull request.

LukyVj committed:
Fix #6 and add myself to the footer :D

Haroenv commented:
Looks good on Chrome, still needs a fix on Safari

![](https://i.imgur.com/RZKZojw.png)

Haroenv committed:
searchbox

Make the cancel button visible
Tweak selector to only show the icon when no input is given
